src/App/component/Login.jsx

```diff
@@ -29,24 +29,37 @@ export default function Login() {
   const [username, setUsername] = useState("")
   const [password, setPassword] = useState("")
 
-
-  const login = () => {
+  const login = async () => {
     if (username === "" || password === "") {
       alert("不準啦馬的>///<")
     } else {
       let payload = {
         username: username,
         password: password
       }
-      Axios.post(`http://localhost:9100/pvs-api/auth/login`, payload)
-        .then((response) => {
-          localStorage.setItem("jwtToken", response.data)
-          goToSelect()
-        })
-        .catch((e) => {
-          alert(e.response.status)
-          console.error(e)
-        })
+      localStorage.setItem("jwtToken", await getJWToken(payload))
+      localStorage.setItem("memberId", await getMemberId())
+      goToSelect()
+    }
+  }
+
+  const getJWToken = async (payload) => {
+    try {
+      const response = await Axios.post(`http://localhost:9100/pvs-api/auth/login`, payload)
+      return response.data
+    } catch (e) {
+      alert(e.response?.status)
+      console.error(e)
+    }
+  }
+
+  const getMemberId = async () => {
+    try {
+      const response = await Axios.get(`http://localhost:9100/pvs-api/auth/memberId?username=${username}`)
```

Comment:
Since `axios` has already implemented the request parameter field in `get` function for us, we may not need to write the combined url ourselves.

Please refer to [this documentation](https://github.com/axios/axios) for more details.

And also in Javascript ES6 standard, if a object's key name is as same as its value's reference name, then this two thing can be combined into only one name, for example:

```js
const foo = 'bar'

const bar = { foo }
```
is equal to 

```js
const foo = 'bar'

const bar =  { 'foo': foo }
```

```suggestion
      const response = await Axios.get('http://localhost:9100/pvs-api/auth/memberId', {
        params: {
          username
        }
      })
```